### PR TITLE
[13.0][MIG] hr_employee_social_media

### PR DESCRIPTION
**Pull request by Saran440**
- field `Social Media Accounts` will show inside `Work Information` tab

### CODE REVIEW

**Comment by jarroyomorales** [comment]
@Saran440 Wouldnt `Private Information`'s tab be a better place for this?

File: hr_employee_social_media/views/hr_employee_view.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <field name="model">hr.employee</field>
         <field name="inherit_id" ref="hr.view_employee_form" />
         <field name="arch" type="xml">
-            <xpath expr="//page[@name='public']/group" position="inside">
+            <xpath expr="//div[@id='o_work_employee_main']" position="inside">
```
Comment (alexey-pelykh) [inline hr_employee_social_media/views/hr_employee_view.xml:8]: The previous place is no longer there?

**Comment by alexey-pelykh** [comment]
@jarroyomorales that would be considered an improvement and anyhow should be done in a separate PR

File: hr_employee_social_media/views/hr_employee_view.xml (hunk 1 above)
Comment (Saran440) [reply hr_employee_social_media/views/hr_employee_view.xml:8]: The previous place is change group to div.
I don't know which is better pattern between
`<xpath expr="//page[@name='public']/div/div" position="inside">`
and
`<xpath expr="//div[@id='o_work_employee_main']" position="inside">`

**Comment by Saran440** [comment]
@jarroyomorales That's a good idea. I will PR after this pr merge :)

**Review by jarroyomorales — APPROVED** [review]

**Pull request closed without merging**